src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[11];
+        figures[0] = figureSupplier.getDefaultFigure();
+        Random random = new Random();
+        for (int i = 1; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure(random.nextInt(5));
+        }
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+    }
```

Comment:
this needs to be redone
remove random
extract array length (11 in your case) to a private class constant
you need only one for loop here, inside of which - if/else or ternary operator. If index is less then half of array length - then random, else - default